src/main/scala/inox/solvers/unrolling/FunctionTemplates.scala

```diff
@@ -33,12 +33,13 @@ trait FunctionTemplates { self: Templates =>
   object FunctionTemplate {
     private val cache: MutableMap[TypedFunDef, FunctionTemplate] = MutableMap.empty
 
-    def apply(tfd: TypedFunDef): FunctionTemplate = cache.getOrElse(tfd, {
+    def apply(tfd: TypedFunDef, pos: utils.Position): FunctionTemplate = cache.getOrElse(tfd, {
       val body: Expr = timers.solvers.simplify.run { simplifyFormula(tfd.fullBody) }
 
       val tpVars = tfd.tps.flatMap(variableSeq).distinct
       val fdArgs: Seq[Variable] = tfd.params.map(_.toVariable)
-      val call: Expr = tfd.applied(fdArgs)
+      // Note: This position is best-effort, since function templates are cached.
+      val call: Expr = tfd.applied(fdArgs).setPos(pos)
```

Comment:
Is the goal here to ensure that all calls in the dumped blocker graph have positions? If so, I don't think you need to set a position on this call because it should never be part of the blocker graph.

(If possible, I'd like to avoid making positions part of the `FunctionTemplate` API)